ktor-client/ktor-client-okhttp/jvm/src/io/ktor/client/engine/okhttp/OkHttpWebsocketSession.kt

```diff
@@ -70,6 +75,10 @@ internal class OkHttpWebsocketSession(
         }
     }
 
+    @ExperimentalWebSocketExtensionApi
+    override val extensions: List<WebSocketExtension<*>>
+        get() = error("WebSocket extensions are not supported on OkHttp")
```

Comment:
Why not just always return an empty list here instead?

Reply:
Yep, make this